.github/workflows/test.yml

```diff
@@ -34,6 +34,9 @@ jobs:
       - name: Run Lint
         run: yarn lint
 
+      - name: Run Tests
+        run: yarn test --silent
```

Comment:
@andrewplummer I think the tests are failing, i am kind not loving tests for web because they currently are so weak. I am wondering if we should just remove it the tests complete

Reply:
let me take care of this I've moved of enzyme and onto `@testing-library/react` I think it might just be a transitionary period here